src/vm/arm64/cgencpu.h

```diff
@@ -79,6 +79,18 @@ typedef INT64 StackElemType;
 // !! This expression assumes STACK_ELEM_SIZE is a power of 2.
 #define StackElemSize(parmSize) (((parmSize) + STACK_ELEM_SIZE - 1) & ~((ULONG)(STACK_ELEM_SIZE - 1)))
 
+
+#ifdef FEATURE_PAL // TODO-ARM64-WINDOWS Add JIT_Stelem_Ref support
```

Comment:
Could you please open an issue on this?

Reply:
Done.  #13590.  Pankaj will probably look at this.